cloudify_cli/bootstrap/bootstrap.py

```diff
@@ -34,6 +34,8 @@
     MANAGER_USER_RUNTIME_PROPERTY,
     MANAGER_KEY_PATH_RUNTIME_PROPERTY,
     REST_PORT)
+from cloudify_cli.commands import plugins
```

Comment:
it somewhat bothers me that `bootstrap/bootstrap` imports something from `commands`. It introduces potential cyclic imports. If absolutely necessary, considering having this import inlined right before its usage with a proper comments describing that it is imported there to avoid cyclic imports